src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -423,6 +423,34 @@ const put = async (frameworkName, config, rawConfig) => {
   }
 };
 
+const execute = async (frameworkName, executionType) => {
+  // send request to framework controller
+  let response;
+  try {
+    response = await axios({
+      method: 'patch',
+      url: launcherConfig.frameworkPath(encodeName(frameworkName)),
+      headers: {
+        'Content-Type': 'application/merge-patch+json',
+      },
+      data: {
+        spec: {
+          executionType: `${executionType.charAt(0)}${executionType.slice(1).toLowerCase()}`,
```

Comment:
Case:

![image](https://user-images.githubusercontent.com/32826762/61034171-58963780-a3f7-11e9-832e-15740975df51.png)

Reply:
right, this is CamelCase